frontend/test/example_test.dart

```diff
@@ -12,11 +12,24 @@ import 'package:angular2/src/modules/testing/lib/testing.dart';
 import 'package:angular2/angular2.dart';
 import 'package:test/test.dart';
 
-//import 'package:frontend/event_list_component.dart';
+import 'packages/mockito/mockito.dart';
```

Comment:
this is strange, I am currently running without the packages directory entirely.
Even though travis still installs with the packages-dir option.